src/components/pdf/Certificate.tsx

```diff
@@ -71,7 +100,8 @@ export default function Certificate({ donation, person }: Props) {
   const formattedDate = formatDateString(donation.createdAt)
   return (
     <Document title="Дарение">
-      <Page size="A4" style={styles.page}>
```

Comment:
Why do we change the page size from `A4` to `LETTER`?

Reply:
The size is being changed as if document size equals A4, background could not be set because content goes to the next page, instead of staying over background.